tests/e2e/support/api/graph/index.ts

```diff
@@ -1 +1,3 @@
 export { me } from './me'
+export { createUser } from './user'
```

Comment:
can be handled in one export e.g. `export { createUser, deleteUser } from './user'` or even `export * from './user'`